From a615409eb90ba0f6290f38d962c500b0e7644855 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sun, 14 Nov 2010 16:15:18 -0500
Subject: [PATCH] ts_test: fix exit status on success

Make sure we return 0 when we're all done to avoid gcc warnings.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 tests/ts_test.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/tests/ts_test.c b/tests/ts_test.c
index 02aa216..fbbc9b8 100644
--- a/tests/ts_test.c
+++ b/tests/ts_test.c
@@ -214,4 +214,6 @@ int main()
 			break;
 	}
 	close_framebuffer();
+
+	return 0;
 }
-- 
1.7.8.4

